src/components/TagSet/TagSet.stories.mdx

```diff
@@ -0,0 +1,274 @@
+  Description,
+  Meta,
+  Story,
+} from "@storybook/addon-docs";
+import { withDesign } from "storybook-addon-designs";
+
+import TagSet from "./TagSet";
+import { getCategory } from "../../utils/componentCategories";
+import DSProvider from "../../theme/provider";
+
+<Meta
+  title={getCategory("TagSet")}
+  component={TagSet}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=51380%3A41869",
+    },
+    jest: ["TagSet.test.tsx"],
+  }}
+  argTypes={{
+    className: { control: false },
+    id: { control: false },
+    isDismissible: {
+      control: "boolean",
+      table: { defaultValue: { summary: "false" } },
+    },
+    onClick: { control: false },
+    tagSetData: { control: false },
+    type: {
+      control: false,
+      table: { defaultValue: { summary: "filter" } },
+    },
+  }}
+/>
+
+# TagSet
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.2.0`    |
+| Latest            | `1.2.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Explore Component Props](#explore-component-props)
+- [Filter Component Props](#filter-component-props)
+- [Accessibility](#accessibility)
+- [Examples](#examples)
+
+## Overview
+
+<Description of={TagSet} />
+
+## Explore Component Props
+
+An "explore" `TagSet` variant is used to display content that can help a user
+start a search and promote exploration. These tags are not dismissible and should
+be used as links to other pages with the content highlighted on the new page.
+
+Notes:
+
+- The `isDismissible` and `onClick` props are not used in this variant.
+
+<Canvas withToolbar>
+  <Story
+    name="TagSet Explore with Controls"
+    argTypes={{
+      isDismissible: { table: { disable: true } },
+      onClick: { table: { disable: true } },
+      tagSetData: { control: false },
+    }}
+    args={{
+      className: undefined,
+      id: "tagSet-id-explore",
+      isDismissible: undefined,
+      onClick: undefined,
+      tagSetData: [
+        { iconName: "arrow", label: <a href="#">Abbey Road</a> },
+        { iconName: "actionLaunch", label: <a href="#">The Beatles</a> },
+        { iconName: "alertWarningOutline", label: <a href="#">Revolver</a> },
+        {
+          iconName: "check",
+          label: (
+            <a href="#">The Beatles Sgt. Pepper's Lonely Hearts Club Band</a>
+          ),
+        },
+      ],
+      type: "explore",
+    }}
+  >
+    {(args) => <TagSet {...args} />}
+  </Story>
+</Canvas>
+
+<ArgsTable story="TagSet Explore with Controls" />
+
+### Explore `tagSetData` Labels
+
+The `tagSetData` prop is an array of objects that contain `iconName` and `label`
+properties. The `iconName` is the name of the icon to display through the
+Reservoir `Icon` component and are optional. The `label` _MUST_ be a React
+element; typically, links are expected to be used for the "explore" variant.
```

Comment:
It could be a react router Link, nextjs link, a regular anchor, the DS link component, and more.... so it's difficult to type exactly what can be passed.